scripts/core/editor3/components/DraftSpellchecker.jsx

```diff
@@ -0,0 +1,244 @@
+/**
+ * @ngdoc React
+ * @module superdesk.core.editor3
+ * @name DraftSpellchecker
+ * @param {String} language the current spellchecker language
+ * @description check the text and shos the inline spellchecker context menu
+ */
+
+import React from 'react';
+import {InlineSpellcheckMenu} from '.';
+
+/** Custom overrides for "error" style. */
+const styleMap = {
+    ERROR: {
+        borderBottomWidth: '1px',
+        borderBottomStyle: 'dotted',
+        borderBottomColor: 'red'
+    }
+};
+
+export class DraftSpellchecker {
```

Comment:
Is it possible that this file and the `InlineSpellcheckerMenu` should be one and the same?

Conceptually, the spellcheck functionality is nothing but the menu, and the decorator, no? You've used this class only to register callbacks, which we can remove anyway, it will be initialised once you add `<Spellchecker />` into the Editor's JSX